pkg/apimachinery/tests/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This package runs tests against the apimachinery which require a Scheme
```

Comment:
If we provide a reference impl of `ObjectMeta` in k8s.io/apimachinery all these tests could be moved as well, couldn't they? We have `ListMeta` already, but it has a lot of noops. We could move `pkg/api/ObjectMeta`.